comfy_cli/command/distribution.py

```
@@ -0,0 +1,1101 @@
+
+def _freeze_env(python_exe: str) -> str | None:
+    """`pip freeze` for a Python env, falling back to `uv pip freeze` when the
+    env has no pip module (uv-created venvs, common for ComfyUI). Returns the
+    requirements text, or None if both fail."""
+    attempts = [
+        [python_exe, "-m", "pip", "freeze"],
+        ["uv", "pip", "freeze", "--python", python_exe],
+    ]
+    for cmd in attempts:
+        try:
+            r = subprocess.run(cmd, capture_output=True, text=True, timeout=60)
```

Pylint (4.0.6)

[warning] 'subprocess.run' used without explicitly defining the value for 'check'.

(W1510)